packages/react-navigation/src/rum/instrumentation/DdRumReactNavigationTracking.tsx

```diff
@@ -10,7 +10,7 @@ import type {
     EventArg,
     NavigationContainerRef,
     Route
-} from '@react-navigation/native';
+} from '@react-navigation/native-v5';
```

Comment:
shouldn't it be just `/native` here? why do we explicitly say it is `v5` (I was thinking v5/v6 separation is only for dev/tests)?

Reply:
Good catch, I need to look deeper to see if we can manage types for both versions